src/components/InstanceRichTooltipRow.tsx

```diff
@@ -0,0 +1,23 @@
+import type { FC } from "react";
+interface Props {
+  title: string;
+  value: React.ReactNode;
+  valueTitle?: string;
+  className?: string;
+}
+
+export const InstanceRichTooltipRow: FC<Props> = ({
```

Comment:
We should keep this component in the `src/components` folder, but make it generic. I suggest a more generic name of `RichTooltipRow` as it is not instance specific and can be reused once we come up with other rich tooltips.